kde-apps/spectacle/spectacle-23.08.0-r1.ebuild

```diff
@@ -55,7 +55,6 @@ DEPEND="${COMMON_DEPEND}
 "
 RDEPEND="${COMMON_DEPEND}
 	>=dev-qt/qdbus-${QTMIN}:*
-	>=dev-qt/qtmultimedia-${QTMIN}:5[qml]
```

Comment:
This has `qml` USEdep for a reason - see:

$ grep -R -C 2 import.*Multimedia
src/Gui/RecordingView.qml-import QtQuick.Layouts 1.15
src/Gui/RecordingView.qml-import QtQuick.Controls 2.15 as QQC2
src/Gui/RecordingView.qml:import QtMultimedia 5.15

That's not an optional runtime dep in my book

Reply:
Well, it is an unconditional import. If there is no crash there may be one at any point in the future, at the very least it will introduce warnings, or potentially erratic behavior.